.github/workflows/broken_link_check.yml

```
@@ -0,0 +1,22 @@
+name: Broken link check
+on:
+  schedule:
```

- Add this so we can kickoff the job manually from here (https://github.com/alteryx/featuretools/actions)
```suggestion
  workflow_dispatch:
  schedule:
```